src/main/java/org/spburegistry/backend/entity/Clinic.java

```diff
@@ -17,12 +18,19 @@ public class Clinic extends BaseEntity {
     @Column(nullable = false)
     private String name;
 
+    @EqualsAndHashCode.Exclude
     @ManyToOne
     @JoinColumn(name = "faculty_id", nullable = false)
     private Faculty faculty;
 
+    @EqualsAndHashCode.Exclude
+    @Builder.Default
     @ManyToMany(mappedBy = "clinics")
-    private Set<Project> projects;
+    private Set<Project> projects = new HashSet<>();
+
+    public void addProject(Project project) {
+        projects.add(project);
+    }
```

Comment:
We don't need this method. Binding will be done by adding clinics to project.

Reply:
Ok